### PR TITLE
[DOC-550] Voyager 1.8.5 changes for Nov 11

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/yugabyte-voyager/release-notes/

### CODE REVIEW

File: docs/content/preview/yugabyte-voyager/known-issues/postgresql.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Review limitations and implement suggested workarounds to successfully migrate d
 - [XID functions is not supported](#xid-functions-is-not-supported)
 - [REFERENCING clause for triggers](#referencing-clause-for-triggers)
 - [BEFORE ROW triggers on partitioned tables](#before-row-triggers-on-partitioned-tables)
-- [Unsupported query construct: Advisory locks](#unsupported-query-construct-advisory-locks)
-- [Unsupported query construct: System columns](#unsupported-query-construct-system-columns)
-- [Unsupported query construct: XML functions](#unsupported-query-construct-xml-functions)
+- [Advisory locks is not yet implemented](#advisory-locks-is-not-yet-implemented)
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/known-issues/postgresql.md:49]: are not implemented, are not supported

according to earlier PR these links were linked from the code somehow?

File: docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ The payload for each migration phase is described in the following table. Note t
...
+| <br/>[assess-migration](../assess-migration/) |<ul><li> Unsupported features (reported in the assessment report without any object names or SQL statements; only count of object per feature)</li> <li>Unsupported datatypes (reported in the assessment report without any table name or column name; only the datatype)</li> <li>Error message while running the assessment if any </li> <li>Table sizing statistics (reads/writes per second, and size in bytes)</li> <li>Index sizing statistics (reads/writes per second, and size in bytes)</li> <li>Schema summary (reported in the assessment report, without object names for each object type)</li> <li>Source connectivity (whether assessment is run with source connectivity) </li> <li>Unsupported query constructs (reported in the assessment report, without any queries information; only count of queries per construct)</li><li>Migration caveats (reported in the assessment report, without any object name or SQL statements; only count of object per caveat) </li></ul>|
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md:46]:
```suggestion
| <br/>[assess-migration](../assess-migration/) |<ul><li> Unsupported features (reported in the assessment report without any object names or SQL statements; only count of object per feature)</li> <li>Unsupported datatypes (reported in the assessment report without any table name or column name; only the datatype)</li> <li>Error message while running the assessment if any </li> <li>Table sizing statistics (reads/writes per second, and size in bytes)</li> <li>Index sizing statistics (reads/writes per second, and size in bytes)</li> <li>Schema summary (reported in the assessment report, without object names for each object type)</li> <li>Source connectivity (whether assessment is run with source connectivity) </li> <li>Unsupported query constructs (reported in the assessment report without any query information; only count of queries per construct)</li><li>Migration caveats (reported in the assessment report without any object name or SQL statements; only count of object per caveat) </li></ul>|
```

Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md:46]: Migration caveates appears to come before unsupported_query_constructs in the report example; what is the order of this list based on

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 3]
```diff
@@ -13,6 +13,30 @@ type: docs
 
 What follows are the release notes for the YugabyteDB Voyager v1 release series. Content will be added as new notable features and changes are available in the patch releases of the YugabyteDB v1 series.
 
+## v1.8.5 - November 11, 2024
+
+### Enhancements
+
+- The guardrails checks to validate source/target database permissions, verify binary dependencies, and check database version compatibility for PostgreSQL in all voyager commands is now enabled by default.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:20]:
```suggestion
- The guardrail checks to validate source/target database permissions, verify binary dependencies, and check database version compatibility for PostgreSQL in all voyager commands are now enabled by default.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 4]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+- UI/UX improvements in the PostgreSQL permission grant script and new checks are added for replication slots, foreign keys, and triggers in PostgreSQL guardrails.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:21]: UX improvements in a script?
What is this script? It is not referred to elsewhere in YBV docs, at least not using this name.

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 5]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+- Object names are scrollable in the analyze schema HTML report for improved navigation.
+- Added constraint names and their corresponding table names when reporting unsupported features related to deferrable and exclusion constraints.
+- Added reporting for the REFERENCING clause for triggers and BEFORE ROW triggers on partitioned tables in the analyze-schema and assess-migration reports.
+- Added functionality to include documentation links for unsupported query constructs in the assessment report.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:25]:
```suggestion
- Added documentation links for unsupported query constructs in the assessment report.
```

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 6]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+- Standardized the format of data sent to the yugabyted control plane via the assess-migration command, ensuring consistent presentation across various sections of the report, such as Unsupported Features, Unsupported Datatypes, and Unsupported Query Constructs.
+
+### Bug fixes
+
+- Fixed the import-schema DDL parsing issue for functions and procedures, where extra spaces before the DDL caused it to be treated as normal DDL, preventing the PLPGSQL parsing logic from triggering.
+- Fixed an issue which resulted in "token too long" errors in export-data-from-target when log level was set to DEBUG.
+
+### Known issues
+
+- The [assess-migration](../reference/assess-migration/) command will fail if the [pg_stat_statements](../../explore/ysql-language-features/pg-extensions/extension-pgstatstatements/) extension is created in a non-public schema, due to the Unsupported Query Constructs feature.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:35]: Unsupported Query Constructs are a feature? What is this actually referring to

File: docs/content/preview/yugabyte-voyager/release-notes.md [hunk 7]
```diff
@@ -13,6 +13,30 @@ type: docs
...
+To bypass this issue, set the environment variable `REPORT_UNSUPPORTED_QUERY_CONSTRUCTS=false`, which disables the feature and proceeds with the command execution.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/release-notes.md:36]: disables what feature?

**Review by priyanshi-yb — APPROVED** [review]
LGTM

File: docs/content/preview/yugabyte-voyager/release-notes.md (hunk 4 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/release-notes.md:21]: Seems internal to me. @priyanshi-yb - can you clarify?

File: docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md (hunk 2 above)
Comment (aishwarya24) [reply docs/content/preview/yugabyte-voyager/reference/diagnostics-report.md]: changing the order based on the example. @priyanshi-yb - let me know whats the right order.

File: docs/content/preview/yugabyte-voyager/release-notes.md (hunk 7 above)
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/release-notes.md:36]: `Unsupported Query Constructs` feature of `assess-migration`

File: docs/content/preview/yugabyte-voyager/release-notes.md (hunk 6 above)
Comment (priyanshi-yb) [reply docs/content/preview/yugabyte-voyager/release-notes.md:35]: @aishwarya24 if we can put this `Unsupported Query Constructs` in quotes then it should be better. 
@ddhodge its a feature `assess-migration` command provides.

**Pull request merged**